docs/src/administration/cli/api-tokens.md

```diff
@@ -153,19 +167,24 @@ To run a cron only on your production environment, check the environment type as
 
```

Comment:
:lady_beetle: The previous line needs to be changed to match the command that's run below.